pkg/apis/conventions/v1alpha1/clusterpodconvention_validation.go

```diff
@@ -79,6 +79,13 @@ func (s *ClusterPodConventionSpec) Validate() validation.FieldErrors {
 		errs = errs.Also(s.Webhook.Validate().ViaField("webhook"))
 	}
 
+	if s.SelectorTarget != PodTemplateSpecLabels && s.SelectorTarget != PodIntentLabels {
+		errs = errs.Also(validation.FieldErrors{
+			field.Invalid(field.NewPath("selectorTarget"), s.SelectorTarget,
+				"Accepted selector target values are \"PodIntent\" and \"PodTemplateSpec\". The default value is set to \"PodTemplateSpec\""),
```

Comment:
Nit: If you used backticks around the string then you wouldn't have to escape the quotes inside the string.  The code would be a bit easier to read.

```suggestion
				`Accepted selector target values are "PodIntent" and "PodTemplateSpec". The default value is set to "PodTemplateSpec"`),
```